controls/automake_works.rb

```diff
@@ -0,0 +1,28 @@
+title 'Tests to confirm automake works as expected'
+
+plan_origin = ENV['HAB_ORIGIN']
+plan_name = input('plan_name', value: 'automake')
+
+control 'core-plans-automake-works' do
+  impact 1.0
+  title 'Ensure automake works as expected'
+  desc '
+  Verify automake by ensuring (1) its installation directory exists and (2) that
+  it returns the expected version
+  '
+  
+  plan_installation_directory = command("hab pkg path #{plan_origin}/#{plan_name}")
+  describe plan_installation_directory do
+    its('exit_status') { should eq 0 }
+    its('stdout') { should_not be_empty }
```

Comment:
Can you add a check for the stderr being empty?

Reply:
Good suggestion.  Added.

Reply:
Done, thanks.